libbeat/common/json_transform.go

```diff
@@ -0,0 +1,46 @@
+package common
```

Comment:
this file moves helpers and exposes the json specific helpers with generic names. I'd rather see some json parsing utility function/module doing the right thing instead of exposing internal helpers.